chaos/chaos.go

```diff
@@ -82,17 +82,17 @@ func (c *Chaos) verifyExecution(clientset kube.Interface) error {
 	}
 
 	if !enrolled {
-		return fmt.Errorf("%s %s is no longer enrolled in kube-monkey. Skipping\n", c.Victim().Kind(), c.Victim().Name())
+		return fmt.Errorf("%s %s is no longer enrolled in kube-monkey. Skipping", c.Victim().Kind(), c.Victim().Name())
 	}
 
 	// Has the victim been blacklisted since scheduling?
 	if c.Victim().IsBlacklisted() {
-		return fmt.Errorf("%s %s is blacklisted. Skipping\n", c.Victim().Kind(), c.Victim().Name())
+		return fmt.Errorf("%s %s is blacklisted. Skipping", c.Victim().Kind(), c.Victim().Name())
 	}
 
 	// Has the victim been removed from the whitelist since scheduling?
 	if !c.Victim().IsWhitelisted() {
-		return fmt.Errorf("%s %s is not whitelisted. Skipping\n", c.Victim().Kind(), c.Victim().Name())
+		return fmt.Errorf("%s %s is not whitelisted. Skipping", c.Victim().Kind(), c.Victim().Name())
```

Comment:
These need a whitespace delimiter if not newline

Reply:
Do they? I followed the codepath that uses these errors and it seems like they just get sent out to glog 

See: https://github.com/asobti/kube-monkey/blob/aa8fc3a8773e14150f48b54bdb7061f15ffe6e65/kubemonkey/kubemonkey.go#L69